src/services/proxyurl/proxyurl.helper.ts

```diff
@@ -0,0 +1,58 @@
+const forceUseProxy = import.meta.env.VITE_USE_PROXYURL === 'true'
```

Comment:
At first, I put the variables in a store but I wasn't convinced after a few tests because:
- proxyurls vars never change, they depend only on the env the app is launched on
- if it is comming from a store, it means: the helper should only be used inside a vue component OR the helper should be used once the store is fully initialized (with the right values). It can be done using `watch()` in v3, but sometimes the layers are created before the init of the proxyhelper, I have no control on that.

So I made it simple, because these urls never change and they depend on the environment, I put them as environment variables.

I have made some changes on the Proxyhelper to make it clear that initialization is needed only because we need to retrieve environment vars from v3.